docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,66 @@
+<!--[metadata]>
++++
+title = "Swarm mode overlay network security model"
+description = "Docker 1.12 Overlay Network Security Model"
+keywords = ["network, docker, documentation, user guide, multihost, swarm mode", "overlay"]
+[menu.main]
+parent = "smn_networking"
+weight=-2
++++
+<![end-metadata]-->
+
+# Docker swarm mode overlay network security model
+
+Overlay networking for Docker Engine swarm mode comes secure out of the box. The
+swarm nodes exchange overlay network information using a gossip protocol. By
+default the nodes encrypt and authenticate information they exchange via gossip
+using the [AES algorithm](https://en.wikipedia.org/wiki/Galois/Counter_Mode) in
+GCM mode. Manager nodes in the swarm rotate the key used to encrypt gossip data
+every 12 hours.
+
+You can also encrypt data exchanged between containers on different nodes on the
+overlay network. To enable encryption, when you create an overlay network pass
+the `--opt encrypted` flag:
+
+```bash
+$ docker network create --opt encrypted --driver overlay my-multi-host-network
+
+dt0zvqn0saezzinc8a5g4worx
+```
+
+When you enable overlay encryption, Docker creates IPSEC tunnels between all the
```

Comment:
@sfsmithcha this isnt correct. The IPsec tunnels are established between the nodes only on-demand. i.e, only between the nodes where the tasks are scheduled on that particular network.